Classes/extra_functions.py

```diff
@@ -126,6 +127,32 @@ async def send_str_as_file(
 ) -> None:
     with BytesIO(file_data.encode("utf8")) as error_file:
         await channel.send(
-            msg_content,
-            file=discord.File(error_file, filename=filename),
+            msg_content, file=discord.File(error_file, filename=filename)
         )
+
+
+async def clean_shutdown(bot, source):
```

Comment:
This function should take in a source and a person instead, since this way it's kind of hacky by adding more to the string than intended and it may break in the future if the warning is reworded.